go.mod

```diff
@@ -5,50 +5,69 @@ go 1.13
+	github.com/docker/libtrust v0.0.0-20160708172513-aabc10ec26b7 // indirect
 	github.com/fatih/color v1.9.0
+	github.com/garyburd/redigo v1.6.2 // indirect
 	github.com/gertd/go-pluralize v0.1.7
 	github.com/ghodss/yaml v1.0.0
 	github.com/gin-contrib/static v0.0.0-20200815103939-31fb0c56a3d1
 	github.com/gin-gonic/gin v1.6.3
 	github.com/go-logr/logr v0.1.0
+	github.com/gofrs/uuid v3.3.0+incompatible // indirect
 	github.com/google/go-cmp v0.5.2
 	github.com/google/go-github/v32 v32.1.0
 	github.com/gosuri/uitable v0.0.4
 	github.com/jetstack/cert-manager v0.14.3
+	github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0 // indirect
+	github.com/kedacore/keda v1.5.1
 	github.com/mholt/archiver/v3 v3.3.0
 	github.com/oam-dev/trait-injector v0.0.0-20200331033130-0a27b176ffc4
-	github.com/onsi/ginkgo v1.13.0
-	github.com/onsi/gomega v1.10.1
+	github.com/onsi/ginkgo v1.14.1
+	github.com/onsi/gomega v1.10.2
 	github.com/openservicemesh/osm v0.3.0
 	github.com/pkg/errors v0.9.1
 	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
 	github.com/wercker/stern v0.0.0-20190705090245-4fa46dd6987f
-	go.uber.org/zap v1.13.0
+	github.com/yvasiyarov/go-metrics v0.0.0-20150112132944-c25f46c4b940 // indirect
+	github.com/yvasiyarov/gorelic v0.0.7 // indirect
+	github.com/yvasiyarov/newrelic_platform_go v0.0.0-20160601141957-9c099fbc30e9 // indirect
+	go.uber.org/zap v1.15.0
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gotest.tools v2.2.0+incompatible
-	helm.sh/helm/v3 v3.2.4
-	k8s.io/api v0.18.6
-	k8s.io/apiextensions-apiserver v0.18.2
-	k8s.io/apimachinery v0.18.6
-	k8s.io/cli-runtime v0.18.6
+	helm.sh/helm/v3 v3.3.4
+	k8s.io/api v0.18.8
+	k8s.io/apiextensions-apiserver v0.18.8
+	k8s.io/apimachinery v0.18.8
+	k8s.io/cli-runtime v0.18.8
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v1.0.0
-	k8s.io/kube-openapi v0.0.0-20200410145947-bcb3869e6f29 // indirect
-	k8s.io/kubectl v0.18.6 // indirect
-	k8s.io/utils v0.0.0-20200414100711-2df71ebbae66
-	sigs.k8s.io/controller-runtime v0.6.0
+	k8s.io/utils v0.0.0-20200603063816-c1c6865ac451
+	sigs.k8s.io/controller-runtime v0.6.2
 )
 
 replace (
-	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible // https://github.com/kubernetes/client-go/issues/628
+	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible // https://github.com/kubernetes/client-go/issues/628
 	github.com/wercker/stern => github.com/oam-dev/stern v1.13.0-alpha
 	// clint-go had a buggy release, https://github.com/kubernetes/client-go/issues/749
-	k8s.io/client-go => k8s.io/client-go v0.18.6
+	k8s.io/client-go => k8s.io/client-go v0.18.8
+)
+
+// Temporary use the replace due to issue https://github.com/kedacore/keda/discussions/1198
+replace github.com/kedacore/keda => github.com/zzxwill/keda v1.5.1
+
+// fix `make` issue
```

Comment:
what issue?

Reply:
https://github.com/docker/distribution/issues/2406

Reply:
Add a link in go.mod. Thanks.